### PR TITLE
Add connection infos to load

### PR DESCRIPTION
**Pull request by bendaamerahm**

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java [hunk 1]
```diff
@@ -334,14 +336,19 @@ private static void createLine(Network network, String id, String name, String v
...
+                                   int node, double p0, double q0, String feederName1, int feederOrder, ConnectablePosition.Direction direction) {
+        var l = vl.newLoad()
             .setId(id)
             .setName(name)
             .setNode(node)
             .setP0(p0)
             .setQ0(q0)
             .add();
+        l.newExtension(ConnectablePositionAdder.class)
+                .newFeeder1()
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java:348]: use newFeeder() here
newFeeder1, newFeeder2, newFeeder3 are used for two and three windings transformers

File: src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java [hunk 2]
```diff
@@ -334,14 +336,19 @@ private static void createLine(Network network, String id, String name, String v
 
     @SuppressWarnings("SameParameterValue")
     private static void createLoad(VoltageLevel vl, String id, String name,
-                                   int node, double p0, double q0) {
-        vl.newLoad()
+                                   int node, double p0, double q0, String feederName1, int feederOrder, ConnectablePosition.Direction direction) {
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java:339]: Typo: feederName

File: src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java [hunk 3]
```diff
@@ -618,7 +632,7 @@ public void testCreateLoadInNodeBreaker() throws Exception {
         loadCreationInfos.setBusOrBusbarSectionId("notFoundBusbarSection");
         loadCreationInfosJson = objectWriter.writeValueAsString(loadCreationInfos);
         mockMvc.perform(post(uriString, TEST_NETWORK_ID).content(loadCreationInfosJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpectAll(status().is5xxServerError(), content().string(new NetworkModificationException(CREATE_LOAD_ERROR, "Busbar section notFoundBusbarSection not found.").getMessage())).andReturn();
+            .andExpectAll(status().is5xxServerError(), content().string(new NetworkModificationException(CREATE_LOAD_ERROR, "java.lang.NullPointerException").getMessage())).andReturn();
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java:635]: Add a check in your getPosition method to test if network.getBusbarSection return null, and throw an explicit exception with a message like before ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java (hunk 1 above)
Comment (bendaamerahm) [reply src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java]: my bad ,nice catch

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**